packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -3615,5 +3616,281 @@ describe("Runtime", () => {
 				});
 			});
 		});
+
+		// TODO: Update these tests when compatibilityMode API is implemented
```

Comment:
nit: Do you need to check in this TODO to remember?

Reply:
It's not needed, but I like to check in with `TODO:` so it's clear this isn't a permenant state

Reply:
Add a link to an ADO item here? AB#<task number>